Gemfile.lock

```diff
@@ -198,23 +193,23 @@ GEM
     i18n (0.9.5)
       concurrent-ruby (~> 1.0)
     ice_nine (0.11.2)
-    image_processing (1.9.3)
+    image_processing (1.10.0)
       mini_magick (>= 4.9.5, < 5)
       ruby-vips (>= 2.0.13, < 3)
     inherited_resources (1.11.0)
       actionpack (>= 5.0, < 6.1)
       has_scope (~> 0.6)
       railties (>= 5.0, < 6.1)
       responders (>= 2, < 4)
-    jaro_winkler (1.5.3)
+    jaro_winkler (1.5.4)
```

Comment:
### [jaro_winkler](https://github.com/tonytonyjan/jaro_winkler)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.5.3 → 1.5.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/tonytonyjan/jaro_winkler/compare/v1.5.3...v1.5.4).

These are the individual commits:
- (46b9359) [Add the license to the gem for packaging.](https://github.com/tonytonyjan/jaro_winkler/commit/46b93599b8a4e8928450622292eb61f33024247f)
- (914f1b4) [Merge pull request #32 from Eluminae/master](https://github.com/tonytonyjan/jaro_winkler/commit/914f1b4dce15ae11ad417f14e237aa2fcf3ea621)
- (dd98482) [chore: package license file](https://github.com/tonytonyjan/jaro_winkler/commit/dd98482679213c870d7cf74c9939050348d0974c)
- (50990da) [chore(release): bump version to 1.5.4](https://github.com/tonytonyjan/jaro_winkler/commit/50990da5cc02675ed8546a47561bf98d630bb9f9)


</details>